physx.js

```diff
@@ -2230,10 +2232,378 @@ const physxWorker = (() => {
     }
   }
 
+  // AnimationSystem
+
+  w.createAnimationAvatar = (mixerPtr) => {
+    const ptr = Module._createAnimationAvatar(
+      mixerPtr,
+    )
+    return ptr;
+  }
+  w.updateAvatar = (animationAvatarPtr, values) => {
+    values.forEach((value, i) => {
+      scratchStack.f32[i] = value;
+    })
+    Module._updateAvatar(
+      animationAvatarPtr, scratchStack.ptr,
+    )
+  }
+  w.updateAvatarString = (animationAvatarPtr, strings) => {

```

Comment:
What is the purpose of this "string" method?

Reply:
I need transfer `float` and `bool` data such as `idleWalkFactor` `jumpStart` to wasm at once, I use `scratchStack.f32` in `updateAvatar()` to do this.

Also need transfer `string` data such as `useAnimation` `emoteAnimation` to wasm at once, I use `scratchStack.u8` in `updateAvatarString()` to do this.

Ideally to merge them into one call, let `scratchStack.u8` string data follow after `scratchStack.f32` float/bool data. I'll try.